### PR TITLE
Add advanced uninstall method to installation.md

### PR DESCRIPTION
**Pull request by vyanovsky**
While adding "Installer common errors" article, found out that advanced uninstall method needs to be added to installation.md article.

### CODE REVIEW

File: docs/adguard-vpn-for-windows/installation.md [hunk 1]
```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
 2. Click *Start* → *Settings* → *Apps* → *Applications and features*. Find AdGuard VPN in the list, click it and choose *Uninstall*.
 
 3. Open the *Control Panel*, then click *Programs* → *Programs and Features* → *Uninstall a program*. Find AdGuard VPN in the list, right-click it and select *Uninstall*.
+
+### Advanced uninstallation {#advanced}
+
+In case regular uninstall doesn't work for any reason, you can try to use an advanced method. First of all, you need to [download the uninstaller tool](https://cdn.adtidy.org/public/Adguard/tools/Uninstall_Utility.zip) created by our developers. Extract the archive to any folder on your PC and run the **Adguard.UninstallUtility.exe** file, and allow the app to make changes to your device. Then follow the instruction below:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/installation.md:28]: change links to cdn.adguardvpn.com, since this is an AdGuard VPN Knowledge base

Comment (el-termikael) [reply docs/adguard-vpn-for-windows/installation.md:28]: Extract the archive to any folder on your PC and run the **Adguard.UninstallUtility.exe** file, and allow the app to make changes to your device.
—>
Extract the archive to any folder on your PC, run the **Adguard.UninstallUtility.exe** file, and allow the app to make changes to your device.

File: docs/adguard-vpn-for-windows/installation.md [hunk 2]
```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
...
+
+* Choose *Standard uninstall*, ***Delete AdGuard VPN*** and click *Uninstall*.
+
+![Standard uninstall *border](https://cdn.adtidy.org/content/kb/ad_blocker/windows/installation/standard-uninstall.png)
+                                 
+* Wait until uninstall is finished — there will be a string in the window:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/installation.md:34]: * Wait until uninstall is finished — there will be a notification.

File: docs/adguard-vpn-for-windows/installation.md [hunk 3]
```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
...
+`[OK] Uninstall finished`
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/installation.md:35]: I guess it's not necessary to mention the notification

File: docs/adguard-vpn-for-windows/installation.md [hunk 4]
```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
...
+                            
+![Uninstall finished *border](https://cdn.adtidy.org/content/kb/ad_blocker/windows/installation/standard-uninstall-2.png)
+
+> Follow the next steps only if after performing first two steps wasn’t enough for some reason. Usually it can be determined by seeing incorrect reinstall of AdGuard, but we strongly suggest contacting our support before using steps 3-4 of advanced uninstall instruction.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/installation.md:39]: > Follow the next steps only if performing first two steps wasn’t enough for some reason. We strongly recommend contacting our support team before using steps 3–4 of advanced uninstall instruction.

File: docs/adguard-vpn-for-windows/installation.md [hunk 5]
```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
...
+
+* Choose *Advanced uninstall*, ***Delete AdGuard VPN*** and click *Uninstall*.
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/installation.md:41]: why ***?

File: docs/adguard-vpn-for-windows/installation.md [hunk 6]
```diff
@@ -22,3 +22,29 @@ If you decide to remove AdGuard VPN from your computer, use one of three options
...
+
+![Advanced uninstall *border](https://cdn.adtidy.org/content/kb/ad_blocker/windows/installation/advanced-uninstall.png)
+                                 
+* Wait until uninstall is finished — there will be a string in the window:
```
Comment (el-termikael) [inline docs/adguard-vpn-for-windows/installation.md:45]: same as above

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/adguard-vpn-for-windows/installation.md (hunk 5 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/installation.md:41]: To emphasize that you need to choose AdGuard VPN (check the screenshot)

File: docs/adguard-vpn-for-windows/installation.md (hunk 6 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/installation.md:45]: To emphasize that you need to choose AdGuard VPN (check the screenshot)

File: docs/adguard-vpn-for-windows/installation.md (hunk 3 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/installation.md:35]: It's not a notification, it's a string that is displayed at the end of a log (check screenshot)

File: docs/adguard-vpn-for-windows/installation.md (hunk 2 above)
Comment (vyanovsky) [reply docs/adguard-vpn-for-windows/installation.md:34]: It's not a notification, it's a string that is displayed at the end of a log in a window (check screenshot)

File: docs/adguard-vpn-for-windows/installation.md (hunk 5 above)
Comment (el-termikael) [reply docs/adguard-vpn-for-windows/installation.md:41]: I believe ** are enough

**Review by el-termikael — APPROVED** [review]

**Pull request merged**